targets/photos/web/public/index.ejs

```diff
@@ -6,18 +6,6 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Public Cozy Photos</title>
-  <% _.forEach(htmlWebpackPlugin.files.js, function(file) {
```

Comment:
wat

Reply:
I don't even know why that code was here...